From 29a0cb54ef2f71864b4ad3a4cccd42d0a25863f4 Mon Sep 17 00:00:00 2001
From: Ruaridh <rhuairahrighairidh@users.noreply.github.com>
Date: Thu, 2 Jun 2022 03:30:18 +0100
Subject: [PATCH 05/12] update type signature to match forked sdk (#10)

---
 types/account.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/types/account.go b/types/account.go
index abdec11b..d34ccdb3 100644
--- a/types/account.go
+++ b/types/account.go
@@ -45,7 +45,7 @@ type EthAccountI interface {
 
 // ProtoAccount defines the prototype function for BaseAccount used for an
 // AccountKeeper.
-func ProtoAccount() authtypes.AccountI {
+func ProtoAccount(_ int64) authtypes.AccountI {
 	return &EthAccount{
 		BaseAccount: &authtypes.BaseAccount{},
 		CodeHash:    common.BytesToHash(emptyCodeHash).String(),
-- 
2.38.1

